model/bin/make_makefile.sh

```diff
@@ -887,7 +887,17 @@
                prop=
              source="w3parall w3triamd $stx $nlx $btx $is $uostmd"
                  IO='w3iogrmd'
-                aux="constants w3servmd w3arrymd w3dispmd w3gsrumd w3timemd w3nmlgridmd $pdlibyow $memcode" ;;
+                aux="constants w3servmd w3arrymd w3dispmd w3gsrumd w3timemd w3nmlgridmd $pdlibyow $memcode"
+                if [ "$scrip" = 'SCRIP' ]
```

Comment:
Spacing for if statements needs to be fixed.

Reply:
done